okonomiyaki/utils/tests/test_test_data.py

```diff
@@ -0,0 +1,34 @@
+import sys
+import zipfile2
+
+from .. import test_data
+
+if sys.version_info < (2, 7):
+        import unittest2 as unittest
+else:
+        import unittest
+
+
+class TestDummyPythonRuntimes(unittest.TestCase):
+
+    def _get_contents_of_runtime(self, runtime):
+        with zipfile2.ZipFile(runtime) as zp:
+            return zp.namelist()
+
+    def test_pythonw_in_dummy_runtime(self):
+        """
+        Ensure that pythonw.exe is included in all Windows runtimes
+        """
+
+        # Given
+        win_runtimes = [f for f in dir(test_data)
```

Comment:
My suggestion here would be either being more explicit (just hardcoding the list of runtimes here), or have a special variable in `okonomiyaki.utils.test_data` that list all runtimes and do the filtering based on `IRuntimeMetadata.factory_from_path`.